projects/valtimo/zgw/src/lib/modules/documenten-api/components/documenten-api-metadata-modal/documenten-api-metadata-modal.component.ts

```diff
@@ -514,7 +514,7 @@ export class DocumentenApiMetadataModalComponent implements OnInit, OnDestroy {
       else if (ontvangstdatum) this.additionalDocumentDate$.next('received');
       else this.additionalDocumentDate$.next('neither');
 
-      const prefillStatus = status || this.defaultValues.status;
+      const prefillStatus = this.defaultValues.status || status;
```

Comment:
This wont work. With this change, it will always show the default; changing it is then not possible anymore